runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesPlugin.java

```diff
@@ -41,6 +44,24 @@
 	private ItemPricesConfig config;
 	@Inject
 	private ItemPricesOverlay overlay;
+	@Inject
+	private  ItemPricesListener listener;
+	@Inject
+	private KeyManager keyManager;
+
+	boolean showPrices = false;
```

Comment:
The name of this unclear, since it's ignored if `showOnAlt` is off.

Reply:
Renamed showPrices to showPricesWhenAltIsPushed to be more clear.